From 3b9c85d8bb9baeebbc690dab78a1a5f3e8f5f136 Mon Sep 17 00:00:00 2001
From: acpaquette <acpaquette@usgs.gov>
Date: Mon, 8 Dec 2025 11:06:25 -0700
Subject: [PATCH] Fix serial call to use kalasiris instead of pysis

---
 polarmosaics/lidar2cnet.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/polarmosaics/lidar2cnet.py b/polarmosaics/lidar2cnet.py
index 0c1ca44..320dff8 100755
--- a/polarmosaics/lidar2cnet.py
+++ b/polarmosaics/lidar2cnet.py
@@ -105,7 +105,7 @@ def main(args=None):
     target = plio.utils.utils.find_in_dict(label, 'TargetName')
 
     # Compute the ISIS3 serial number
-    serial = isis.getsn(from_=args.cube).decode("utf-8").strip()
+    serial = isis.getsn(from_=arg.cube).stdout.strip()
 
     columns = ['id', 'pointType', 'serialnumber', 'measureType',
                'sample', 'line',
-- 
2.49.0

